test/route.test.ts

```diff
@@ -0,0 +1,159 @@
+import * as sj from '../dist'
```

Comment:
Because the `route` is a function in the swap contract, so let's implement the test inside the `swap.test`